### PR TITLE
EJ2-Documentation-848214-Ug-Documentation-Foreign-Key-Columns

### PR DESCRIPTION
**Pull request by harinikrh**

### CODE REVIEW

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine

File: ej2-javascript/code-snippet/grid/foreign-key-cs2/index.js [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
-        { field: 'Freight', headerText: 'Freight', width: 100, textAlign: 'Right'},
-        { field: 'ShipName', headerText: 'Ship Name', width: 180 }
-    ],
-    height: 315
+        destroy: function () {
+          // to destroy the custom component.
+          autoCompleteObj.destroy();
+        },
+        write: function (args) {
+          // to show the value for date picker
```
Comment (VigneshNatarajan27) [inline ej2-javascript/code-snippet/grid/foreign-key-cs2/index.js:29]: change date picker autocomplete

File: ej2-javascript/code-snippet/grid/foreign-key-cs2/index.js [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
+  toolbar: ['Add', 'Edit', 'Delete', 'Update', 'Cancel'],
+  editSettings: { allowEditing: true },
+  columns: [
+    { field: 'OrderID', headerText: 'Order ID', isPrimaryKey: true, textAlign: 'Right', width: 100 },
+    {
+      field: 'EmployeeID', headerText: 'Employee Name', width: 150, foreignKeyValue: 'FirstName', dataSource: employeeData,
+      edit: {
+        create: function () {
+          // to create input element
+
```
Comment (VigneshNatarajan27) [inline ej2-javascript/code-snippet/grid/foreign-key-cs2/index.js:14]: remove unwanted empty line

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**